src/spatio_temporal_voxel_grid.cpp

```diff
@@ -132,12 +132,15 @@ void SpatioTemporalVoxelGrid::ClearFrustums(const \
     }
     else if (it->_model_type == THREE_DIMENSIONAL_LIDAR)
     {
-      frustum = new geometry::ThreeDimensionalLidarFrustum( \
-                                                    it->_vertical_fov_in_rad,
-                                                    it->_vertical_fov_padding_in_m,
-                                                    it->_horizontal_fov_in_rad,
-                                                    it->_min_z_in_m,
-                                                    it->_max_z_in_m);
+        frustum = new geometry::ThreeDimensionalLidarFrustum( \
+                                                      it->_vertical_fov_in_rad,
```

Comment:
Misformating

Reply:
I'm sorry, I'm not sure what the problem is here, could you please answer it? Or suggest a change